test/WorkerMgt.t.sol

```diff
@@ -20,7 +19,7 @@ contract WorkerMgtTest is Test {
     IBLSApkRegistry.PubkeyRegistrationParams[] pubkeys;
 
     function setUp() public {
-        PADORegistryCoordinator registryCoordinator = PADORegistryCoordinator(
```

Comment:
The test should use PADORegistryCoordinator.

Reply:
fixed